### PR TITLE
Upgrade RHTAP e2e go version to 1.20

### PR DESCRIPTION
**Pull request by flacatus**

### CODE REVIEW

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e

File: ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ build_root:
   image_stream_tag:
     name: builder
     namespace: ocp
-    tag: rhel-9-golang-1.19-openshift-4.13
+    tag: rhel-9-golang-1.20-openshift-4.13
```
Comment (psturc) [inline ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml:5]:
```suggestion
    tag: rhel-9-golang-1.20-openshift-4.14
```

**Comment by flacatus** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e

**Review by davidmogar — APPROVED** [review]

**Pull request merged**